todo_app/app.py

```diff
@@ -1,5 +1,5 @@
 from flask import Flask, render_template, request, redirect
-from todo_app.data import trello_items
+from todo_app.data import mongo_items
```

Comment:
Nice, the fact this is such a simple change here is a good sign that you designed this effectively all the way back in the Trello exercise 😄